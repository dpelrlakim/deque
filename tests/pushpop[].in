newDefault dq1
push_front dq1 0
[] dq1 0
push_front dq1 -1
at dq1 0
push_front dq1 2
[] dq1 0
push_front dq1 -3
at dq1 0
push_front dq1 4
[] dq1 0
push_front dq1 -5
at dq1 0
[] dq1 5
at dq1 6
pop_front dq1
pop_front dq1
pop_front dq1
pop_front dq1
pop_front dq1
pop_front dq1
newDefault dq1
push_back dq1 0
at dq1 0
push_back dq1 1
[] dq1 1
push_back dq1 -2
at dq1 2
push_back dq1 2147483647
[] dq1 3
push_back dq1 -2147483648
at dq1 4
push_back dq1 300
[] dq1 5
pop_back dq1
pop_back dq1
pop_back dq1
pop_back dq1
pop_back dq1
pop_back dq1
at dq1 0
