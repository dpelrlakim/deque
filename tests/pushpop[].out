dq1:
dq1: 0
0
dq1: 0
dq1: -1 0
-1
dq1: -1 0
dq1: 2 -1 0
2
dq1: 2 -1 0
dq1: -3 2 -1 0
-3
dq1: -3 2 -1 0
dq1: 4 -3 2 -1 0
4
dq1: 4 -3 2 -1 0
dq1: -5 4 -3 2 -1 0
-5
dq1: -5 4 -3 2 -1 0
0
dq1: -5 4 -3 2 -1 0
Exception
dq1: -5 4 -3 2 -1 0
dq1: 4 -3 2 -1 0
dq1: -3 2 -1 0
dq1: 2 -1 0
dq1: -1 0
dq1: 0
dq1:
dq1:
dq1: 0
0
dq1: 0
dq1: 0 1
1
dq1: 0 1
dq1: 0 1 -2
-2
dq1: 0 1 -2
dq1: 0 1 -2 2147483647
2147483647
dq1: 0 1 -2 2147483647
dq1: 0 1 -2 2147483647 -2147483648
-2147483648
dq1: 0 1 -2 2147483647 -2147483648
dq1: 0 1 -2 2147483647 -2147483648 300
300
dq1: 0 1 -2 2147483647 -2147483648 300
dq1: 0 1 -2 2147483647 -2147483648
dq1: 0 1 -2 2147483647
dq1: 0 1 -2
dq1: 0 1
dq1: 0
dq1:
Exception
dq1:
